src/bp-forums/templates/default/js/editor.js

```diff
@@ -84,17 +84,6 @@ jQuery( document ).ready(
 						}
 					);
 
-					element.addEventListener( 'paste', function ( event ) {
```

Comment:
@mahedihasannoman Can you check in the git commit history. Why was this code added in past? I want to make sure we are not missing any other case by removing this code.

Reply:
Hi @dipskakadiya , This code was added in order to - Enabled copy-paste HTML for forum discussion, edit forum discussion but this code is actually causing this problem. It is already happening by the medium editor package. See the commit: https://github.com/buddyboss/buddyboss-platform/pull/2699/commits/ddf5084e6529fb2e10e300f9a7714b2323735c2b